packages/desktop-electron/package.json

```diff
@@ -7,10 +7,12 @@
   "scripts": {
     "clean": "rm -rf dist",
     "update-client": "bin/update-client",
-    "build": "electron-builder",
-    "watch": "cross-env ACTUAL_DOCUMENT_DIR=\"../../data\" ACTUAL_DATA_DIR=\"../../data\" electron ."
+    "build": "yarn build:dist && electron-builder",
+    "build:dist": "rm -rf build && tsc --p tsconfig.dist.json && yarn copy-assets",
+    "copy-assets": "copyfiles --exclude 'client-build/**/*' **/*.html build",
```

Comment:
tsc compile only copies over the compiled js. We need to manually copy over the html that electron requires as it isnt' referenced in the js.